test/conformance/helpers/broker_data_plane_test_helper.go

```diff
@@ -219,8 +219,31 @@ func BrokerV1Beta1ConsumerDataPlaneTestHelper(
 	secondTriggerName := "second-trigger"
 	loggerName := "logger-pod"
 	secondLoggerName := "second-logger-pod"
+	transformerName := "transformer-pod"
+	replySource := "origin-for-reply"
+	eventID := "consumer-broker-tests"
+	baseSource := "consumer-test-sender"
 	eventTracker, _ := recordevents.StartEventRecordOrFail(client, loggerName)
 	secondTracker, _ := recordevents.StartEventRecordOrFail(client, secondLoggerName)
+
+	baseEvent := ce.NewEvent()
+	baseEvent.SetID(eventID)
+	baseEvent.SetType(testlib.DefaultEventType)
+	baseEvent.SetSource(baseSource)
+	baseEvent.SetSpecVersion("1.0")
+	body := fmt.Sprintf(`{"msg":%q}`, eventID)
+	if err := baseEvent.SetData(ce.ApplicationJSON, []byte(body)); err != nil {
+		t.Fatalf("Cannot set the payload of the baseEvent: %s", err.Error())
+	}
+
+	transformMsg := []byte(`{"msg":"Transformed!"}`)
+	transformPod := resources.EventTransformationPod(
+		transformerName,
+		"reply-check-type",
+		"reply-check-source",
+		transformMsg,
+	)
+	client.CreatePodOrFail(transformPod, testlib.WithService(transformerName))
```

Comment:
I wonder if we should've waited here for endpoints ready instead of in the test, but I see you do it in the test, let's see if that helps. Thanks!